book/deques/deque.py

```diff
@@ -0,0 +1,21 @@
+class Deque:
+    def __init__(self):
+        self._items = []
+
+    def is_empty(self):
+        return self._items == []
+
+    def add_front(self, item):
+        self._items.append(item)
+
+    def add_rear(self, item):
+        self._items.insert(0,item)
```

Comment:
Open to including this too, since you're updating this anyway?

```suggestion
        self._items.insert(0, item)
```